src/ReflectionClass.php

```diff
@@ -62,11 +72,8 @@ public static function collectInterfacesFromClassNode(ClassLike $classLikeNode)
         if ($implementsList) {
             foreach ($implementsList as $implementNode) {
                 if ($implementNode instanceof FullyQualified) {
-                    $implementName  = $implementNode->toString();
-                    $interface      = interface_exists($implementName, false)
-                        ? new parent($implementName)
-                        : new static($implementName);
-                    $interfaces[$implementName] = $interface;
+                    $implementName              = $implementNode->toString();
+                    $interfaces[$implementName] = new static($implementName);
```

Comment:
1. It used to be needed, because `Go\ParserReflection\ReflectionClass` didn't previously support built in classes (or interfaces in this case), but now it does.
1. This new situation **should** already be under test, but I will verify: :memo: